frontend/src/layout/navigation-3000/Navigation.scss

```diff
@@ -77,7 +72,7 @@
         }
 
         li + li {
-            margin-top: 0.25rem;
+            margin-top: -1px;
```

Comment:
Sorry for the late review here but looks like this is causing nav buttons to overlap
![overlap](https://github.com/PostHog/posthog/assets/6685876/d99f1c1c-3fba-4db4-8232-8a37798fd1a5)